test/mock_windows.uts

```diff
@@ -147,3 +147,17 @@ emulate_main_input(data)
 from ctypes import wintypes
 data = ["#AUTOCOMPLETE{scapy.config.conf.vers}", "assert _ == scapy.config.conf.version"]
 emulate_main_input(data)
+
+############
+############
++ Windows arch unit tests
+
+
+= show_interfaces
+
+from scapy.arch import *
+show_interfaces()
```

Comment:
Could you check the output of this function ? When we can do it, I like when we mock instead of calling a function. That gives us a better value for code coverage.